pandas/core/dtypes/dtypes.py

```diff
@@ -316,7 +315,10 @@ def _validate_categories(categories, fastpath=False):
         from pandas import Index
 
         if not isinstance(categories, ABCIndexClass):
-            categories = Index(categories)
+            categories = Index(categories, tupleize_cols=False)
+
```

Comment:
should be an elif and instead should use is_categorical

Reply:
Isn't the point of the ABC ones to avoid imports from itself ? (as `is_categorical` uses CategoricalDtype)

Reply:
@jreback Can you answer this one? No problem to change it if you think it is better, but it seems to me using `isinstance(categories, ABCCategoricalIndex)` is consistent with the use of `isinstance(categories, ABCIndexClass)` just above (also in readability of the code)

Reply:
hmm, yeah I guess you have to do this. but you can also check ``isinstance(categories, (ABCCateoricalIndex, ABCCategorical))`` is more generic.

Reply:
updated to use `isinstance(categories, (ABCCateoricalIndex, ABCCategorical))` first and then only to check for Index